cmd/migrate/executor/cstor_pool.go

```diff
@@ -87,6 +87,8 @@ func (m *MigrateOptions) RunCStorSPCMigrate() error {
 		return errors.Errorf("Failed to migrate cStor SPC : %s", m.spcName)
 	}
 	klog.Infof("Successfully migrated spc %s to cspc", m.spcName)
-
+	klog.Infof("Make sure to migrate the associated PVCs, " +
+		"to find the old PVCs use `kubectl get cstorvolume.openebs.io -n <openebs-namespace>`, " +
```

Comment:
```suggestion
		"to list CStorVolume for the PVs which are pending migration use `kubectl get cstorvolume.openebs.io -n %s -l=openebs.io/storage-pool-claim=%s`, " +
```
So the command can be copy pasted.